assets/js/base/components/cart-checkout/address-form/address-form.tsx

```diff
@@ -1,7 +1,7 @@
 /**
```

Comment:
Removed lots of duplication in this file, and moved custom formatting to a callback instead of handling it inline.